site/content/blog/2022-03-01-whats-new-in-svelte-march-2022.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "What's new in Svelte: March 2022"
+description: "Shadow endpoints and update detection come to SvelteKit"
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+Some long-requested features have been added to SvelteKit this month... including shadow endpoints! This change in how `load` functions makes it easier to fetch data required for basic pages, redirect from POST responses and handle 404s and other errors.
+
```

Comment:
There's a new Svelte Summit coming up in April that I think makes sense to mention here (though I am a bit biased! 😅). As per usual the website is here: https://sveltesummit.com